.github/CODEOWNERS

```diff
@@ -299,10 +299,10 @@ testbed/mockdatasenders/mockdatadogagentexporter/                   @open-teleme
 # List of distribution maintainers for OpenTelemetry Collector Contrib
 #
 #####################################################
-reports/distributions/core.yaml    @open-telemetry/collector-contrib-approvers 
-reports/distributions/contrib.yaml @open-telemetry/collector-contrib-approvers 
+reports/distributions/core.yaml    @open-telemetry/collector-contrib-approvers
+reports/distributions/contrib.yaml @open-telemetry/collector-contrib-approvers
 
 
 ## UNMAINTAINED components
 
-receiver/googlecloudspannerreceiver/                                @open-telemetry/collector-contrib-approvers 
+receiver/googlecloudspannerreceiver/                                @open-telemetry/collector-contrib-approvers
```

Comment:
Please revert those white space changes, this file is generated

Reply:
Will do. FYI, this PR is just a reference. I've been keeping it up to date as I split off smaller PRs and they are integrated.